concent_api/core/tests/utils.py

```diff
@@ -1036,3 +1035,19 @@ def _get_verification_deadline_as_timestamp(
     def _get_uuid(last_char: Optional[str] = None) -> str:
         assert last_char is None or (isinstance(last_char, str) and len(last_char) == 1)
         return generate_uuid(last_char)
+
+    def send_request(
+        self,
+        url: str,
+        data: bytes,
+        content_type: Optional[str] = 'application/octet-stream',
```

Comment:
why is it `Optional`? Can it be set to `None` and still make sense?
The same applies for `golem_messages_version`

Reply:
It can be None or some other value if you want test it. Default values are given, but if you pass None explicitly (by function call) the value will be None. And in 99.9% cases you don't have to pass anything and default data is correct data. I think it is good behavior

Reply:
is there any case where it is different than `'application/octet-stream'`? If not change it.